src/components/sensitivity-analysis-result-tab.js

```diff
@@ -9,20 +9,319 @@ import { useNodeData } from './study-container';
+            </Tabs>
+            {pagedSensitivityResult}
+        </>
+    );
+};
+
+const DATA_KEY_TO_FILTER_KEY = {
+    funcId: 'functionIds',
+    varId: 'variableIds',
+    contingencyId: 'contingencyIds',
+};
+
+const DATA_KEY_TO_SORT_KEY = {
+    funcId: 'FUNCTION',
+    varId: 'VARIABLE',
+    contingencyId: 'CONTINGENCY',
+    functionReference: 'REFERENCE',
+    value: 'SENSITIVITY',
+    functionReferenceAfter: 'POST_REFERENCE',
+    valueAfter: 'POST_SENSITIVITY',
+};
+
+function addIndexerParamsToSelector(indexer, selector) {
+    if (!indexer) return;
+
+    const sortKeysWithWeightAndDirection = {};
+    const sortingAsKeyAndDirSign = indexer.getSortingAsKeyAndCodedRank();
+    sortingAsKeyAndDirSign.forEach(([k, v]) => {
+        const sortKey = DATA_KEY_TO_SORT_KEY[k];
+        if (sortKey) sortKeysWithWeightAndDirection[sortKey] = v;
+    });
+    if (Object.keys(sortKeysWithWeightAndDirection).length) {
+        selector.sortKeysWithWeightAndDirection =
+            sortKeysWithWeightAndDirection;
+    }
+    const userFiltering = indexer.getUserFiltering();
+    Object.entries(userFiltering).forEach(([k, v]) => {
+        const filterKey = DATA_KEY_TO_FILTER_KEY[k];
+        if (filterKey) selector[filterKey] = v;
+    });
+}
+
+const tellDiff = (what, was, now) => {
+    if (was !== now) {
+        console.debug(what + ' has changed', was, now);
+        return true;
+    }
+    return false;
+};
+
+function PagedSensitivityResult({
+    nOrNkIndex,
+    sensiKindIndex,
+    studyUuid,
+    nodeUuid,
+}) {
+    const [userRowsPerPage, setUserRowsPerPage] = useState(DEFAULT_PAGE_COUNT);
+    const [page, setPage] = useState(0);
+    const [version, setVersion] = useState();
+    const [overAllCount, setOverAllCount] = useState(null);
+    const [filteredCount, setFilteredCount] = useState(null);
+
+    const synthRef = useRef();
+    const [next, prev] = [{}, synthRef.current];
+    next.sensiKindIndex = sensiKindIndex;
+    next.nOrNkIndex = nOrNkIndex;
+    next.nodeUuid = nodeUuid;
+    next.indexer = prev?.indexer;
+    next.askedFilterVersion = prev?.askedFilterVersion;
+    next.page = prev?.page;
+    next.userRowsPerPage = prev?.userRowsPerPage;
+    next.version = prev?.version;
+
+    if (
+        tellDiff('node', prev?.nodeUuid, nodeUuid) ||
+        tellDiff('sensi kind ', prev?.sensiKindIndex, sensiKindIndex) ||
+        tellDiff('pre/post aleas', prev?.nOrNkIndex, nOrNkIndex)
+    ) {
+        setUserRowsPerPage(DEFAULT_PAGE_COUNT);
+        setPage(0);
+        setOverAllCount(null);
+        setFilteredCount(null);
+        setVersion(0);
+
+        const pars = [true, false, null, setVersion];
+        const rowIndexer = new KeyedColumnsRowIndexer(...pars);
+        const colKey = nOrNkIndex === 0 ? 'value' : 'valueAfter';
+        const changeWay = CHANGE_WAYS.SIMPLE;
+        rowIndexer.updateSortingFromUser(colKey, changeWay);
+        rowIndexer.updateSortingFromUser(colKey, changeWay);
```

Comment:
No, in fact it is on purpose. I just used the user interaction way to set main sort key to colKey from highest on top to lowest value at bottom.
Without the repetition, it would sort from lowest value on top to highest value at bottom.

Reply:
Done